islandora_transforms/RELS-EXT_to_solr.xslt

```diff
@@ -58,14 +58,17 @@
     <xsl:template match="*[normalize-space(.)]" mode="rels_ext_element">
       <xsl:param name="prefix"/>
       <xsl:param name="suffix"/>
-
-      <xsl:call-template name="rels_ext_fields">
-        <xsl:with-param name="prefix" select="$prefix"/>
-        <xsl:with-param name="suffix" select="$suffix"/>
-        <xsl:with-param name="type">literal</xsl:with-param>
-        <xsl:with-param name="value" select="text()"/>
-      </xsl:call-template>
-    </xsl:template>
+      <!-- XXX: Parameters as passed in are strings... Let's deal with it as a
+        string here, for convenience. -->
+      <xsl:if test="string($index_compound_sequence) = 'true' or (string($index_compound_sequence) = 'false' and (not(self::islandora-rels-ext:isSequenceNumberOf) and  not(self::islandora-rels-ext:* and starts-with(local-name(), 'isSequenceNumberOf')))">
```

Comment:
I don't believe the first bit should ever have an effect... the test for `self::islandora-rels-ext:isSequenceNumberOf`... I don't think we'll ever have a relationship of the like, without the PID bit on the end... which is accounted for with the other bit.
```suggestion
      <xsl:if test="string($index_compound_sequence) = 'true' or (string($index_compound_sequence) = 'false' and not(self::islandora-rels-ext:* and starts-with(local-name(), 'isSequenceNumberOf')))">
```